examples/c/src/main.cpp

```diff
@@ -9,25 +12,37 @@ void CXX_API(const char* model_path) {
   auto model = OgaModel::Create(model_path);
   std::cout << "Creating tokenizer..." << std::endl;
   auto tokenizer = OgaTokenizer::Create(*model);
+  auto tokenizer_stream = OgaTokenizerStream::Create(*tokenizer);
+
+  while (true) {
+    std::string text;
+    std::cout << "Prompt: " << std::endl;
+    std::getline(std::cin, text);
+
+    const std::string prompt = "<|user|>" + text + "<|end|><|assistant|>";
```

Comment:
The [system prompt](https://huggingface.co/microsoft/Phi-3-mini-4k-instruct#chat-format) includes some newline characters. Can we add those?